src/MixpanelEventForwarder.js

```diff
@@ -62,13 +62,79 @@ var constructor = function () {
             if (!testMode) {
                 renderSnippet();
             }
-            mixpanel.init(
-                settings.token,
+            // Build init options object
+            var initOptions = {
+                api_host: forwarderSettings.baseUrl,
+            };
+
+            // Session Replay boolean settings
+            var boolSettings = [
+                { key: 'recordHeatmapData', mappedKey: 'record_heatmap_data' },
+                { key: 'autocapture', mappedKey: 'autocapture' },
+                { key: 'recordCanvas', mappedKey: 'record_canvas' },
+            ];
+
+            // Session Replay numeric settings
+            var numericSettings = [
                 {
-                    api_host: forwarderSettings.baseUrl,
+                    key: 'recordSessionsPercent',
+                    mappedKey: 'record_sessions_percent',
                 },
-                'mparticle'
-            );
+                {
+                    key: 'recordIdleTimeoutMs',
+                    mappedKey: 'record_idle_timeout_ms',
+                },
+                { key: 'recordMaxMs', mappedKey: 'record_max_ms' },
+                { key: 'recordMinMs', mappedKey: 'record_min_ms' },
+            ];
+
+            // Session Replay string settings
+            var stringSettings = [
+                {
+                    key: 'recordMaskTextSelector',
+                    mappedKey: 'record_mask_text_selector',
+                },
+                {
+                    key: 'recordBlockSelector',
+                    mappedKey: 'record_block_selector',
+                },
+                { key: 'recordBlockClass', mappedKey: 'record_block_class' },
+                {
+                    key: 'recordMaskTextClass',
+                    mappedKey: 'record_mask_text_class',
+                },
+            ];
+
+            // Process boolean settings
+            boolSettings.forEach(function (setting) {
+                if (forwarderSettings[setting.key] != null) {
+                    initOptions[setting.mappedKey] =
+                        forwarderSettings[setting.key] === 'True';
+                }
+            });
+
+            // Process numeric settings
+            numericSettings.forEach(function (setting) {
+                if (forwarderSettings[setting.key] != null) {
+                    var numericValue = parseInt(
+                        forwarderSettings[setting.key],
+                        10
+                    );
+                    if (!isNaN(numericValue)) {
+                        initOptions[setting.mappedKey] = numericValue;
+                    }
```

Comment:
Extra credit: Maybe we can extract this logic into a utility function to improve readability.

```
function parseIntSafe(value) {
    var n = parseInt(value, 10);
    return isNaN(n) ? undefined : n;
}
```